x/slashing/types/types.proto

```diff
@@ -0,0 +1,35 @@
+syntax = "proto3";
+package cosmos_sdk.x.slashing.v1;
+
+option go_package = "types";
```

Comment:
I think some other modules do this wrong and need to be fixed. This should be `github.com/cosmos/cosmos-sdk/x/slashing/types`.